src/amo/api/collections.js

```diff
@@ -28,42 +32,153 @@ export const getCollectionDetail = (
 
 type GetCollectionAddonsParams = {|
   ...GetCollectionParams,
+  nextURL?: string,
   page?: number,
 |};
 
 export const getCollectionAddons = (
-  { api, page, slug, user }: GetCollectionAddonsParams
-) => {
+  { api, nextURL, page, slug, user }: GetCollectionAddonsParams
+): Promise<PaginatedApiResponse<ExternalCollectionAddon>> => {
   if (!slug) {
     throw new Error('slug is required');
   }
   if (!user) {
     throw new Error('user is required');
   }
 
-  return callApi({
+  const request = {
     auth: true,
-    endpoint: `accounts/account/${user}/collections/${slug}/addons`,
-    params: { page },
+    endpoint:
+      nextURL || `accounts/account/${user}/collections/${slug}/addons`,
+    params: undefined,
     state: api,
-  });
+  };
+  if (!nextURL || page) {
```

Comment:
This seems an odd conditional. I'd think you'd want to check for either `!nextURL && page` (to make sure the param we're setting has a value below) or just check for a false-y `nextURL`... I guess I just find it strange but maybe there's a reason I'm missing here. 😄